detection_rules/devtools.py

```diff
@@ -1088,3 +1089,16 @@ def get_branches(outfile: Path):
     branch_list = get_stack_versions(drop_patch=True)
     target_branches = json.dumps(branch_list[:-1]) + "\n"
     outfile.write_text(target_branches)
+
+
+@dev_group.group('integrations')
+def integrations_group():
+    """Commands for dev integrations methods."""
+
+
+@integrations_group.command('build-integration-manifests')
+@click.option('--overwrite', '-o', is_flag=True, help="Overwrite the existing integrations-manifest.json.gz file")
+def build_integration_manifests(overwrite: bool):
+    """Builds integrations manifests file"""
+    github_token = get_github_token()
```

Comment:
Better to expose it as an option, like the other commands

```suggestion
@click.option("--token", required=True, prompt=get_github_token() is None, default=get_github_token(),
              help="GitHub token to use for the PR", hide_input=True)
def build_integration_manifests(overwrite: bool, token: str):
    """Builds integrations manifests file"""
```